src/main/java/org/wiremock/extension/jwt/JwtHelpersExtension.java

```diff
@@ -19,8 +19,8 @@ public Map<String, Helper<?>> provideTemplateHelpers() {
         return Map.of(
                 "jwt", jwtHandlebarsHelper,
                 "claims", new ClaimListHandlebarsHelper(),
-                "jwks", jwksHandlebarsHelper
-        );
+                "object", new ObjectHandlebarsHelper(),
```

Comment:
Suggest we change this to `claimsObject` to reduce the likelihood of future name clashes.